kafka-connector/src/main/java/com/google/pubsub/kafka/common/ConnectorUtils.java

```diff
@@ -43,6 +43,7 @@
   public static final String CPS_MESSAGE_KEY_ATTRIBUTE = "key";
   public static final String CPS_MESSAGE_PARTITION_ATTRIBUTE = "kafka_partition";
   public static final String CPS_MESSAGE_KAFKA_TOPIC_ATTRIBUTE = "kafka_topic";
+  public static final String CPS_MESSAGE_KAFKA_STRUCT_ATTRIBUTE = "message";
```

Comment:
The naming scheme here is a bit inconsistent. For the above declarations, CPS_MESSAGE_***_ATTRIBUTE means the attribute name in the CPS message, but this means the attribute name in the Kafka struct for the CPS message. I think it'd be better if whichever side's attribute the string is the key for is always either first or last.